### PR TITLE
update oss-prow, add more plugins

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @cjwagner 

cc @chrislovecnm

### CODE REVIEW

**Comment by krzyzacy** [comment]
also let me add a presubmit job :-)

File: prow/plugins.yaml [hunk 1]
```diff
@@ -12,170 +12,15 @@
...
-  - approve
-  - assign
-  - cat
-  - dog
-  - lgtm
-  - owners-label
-  - size
-  - trigger 
-  GoogleCloudPlatform/gke-tracing-demo:
+  GoogleCloudPlatform:
```
Comment (cjwagner) [inline prow/plugins.yaml]: Is this safe to do? Do we receive webhooks for all repos under `GoogleCloudPlatform` (org scoped webhook)? If so this probably isn't acceptable.

File: prow/plugins.yaml [hunk 2]
```diff
@@ -184,21 +29,13 @@ plugins:
...
-  - size
-  - trigger
+  - blunderbuss
+  - config-updater
+  - golint
+  - heart
+  - pony
+  - shrug
+  - wip
+  - yuks
```
Comment (cjwagner) [inline prow/plugins.yaml]: It is unclear why most of these are only enabled for oss-test-infra. In particular `wip`, `blunderbuss`, and `golint`.

File: prow/config.yaml [hunk 3]
```diff
@@ -57,3 +57,31 @@ presubmits:
...
+        - bash
+        - -C
+        - cd
+        - /home/prow/go/src/k8s.io/test-infra
+        - "&&"
+        - bazel
+        - run
+        - prow/cmd/checkconfig
+        - --
+        - --config-path=/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/prow/config.yaml
```
Comment (krzyzacy) [inline prow/config.yaml]: /shrug

File: prow/plugins.yaml (hunk 1 above)
Comment (cjwagner) [reply prow/plugins.yaml]: Nevermind, I just saw: https://github.com/GoogleCloudPlatform/oss-test-infra/pull/4#discussion_r261424809
> we receive webhooks for individual repos

Comment (krzyzacy) [reply prow/plugins.yaml]: we only receive per-repo webhooks afaik

File: prow/config.yaml (hunk 3 above)
Comment (cjwagner) [reply prow/config.yaml]: `--config-path=prow/config.yaml` should be sufficient?

File: prow/plugins.yaml (hunk 2 above)
Comment (krzyzacy) [reply prow/plugins.yaml]: cc @chrislovecnm 

we probably don't want to change existing behavior for other repos now... assume they choose this list of plugins for a reason?

File: prow/plugins.yaml (hunk 1 above)
Comment (cjwagner) [reply prow/plugins.yaml]: If that isn't a certainty we should validate that before merging this so we don't start commenting (and trying to do more) in repos where Prow should not be enabled.

File: prow/config.yaml (hunk 3 above)
Comment (krzyzacy) [reply prow/config.yaml]: that will validate the k8s.io/test-infra config I think... (I have a cd above)

File: prow/plugins.yaml (hunk 1 above)
Comment (krzyzacy) [reply prow/plugins.yaml]: (I don't think either me or @chrislovecnm has org-level webhook access, we only had touched repos we have access to)

File: prow/config.yaml (hunk 3 above)
Comment (krzyzacy) [reply prow/config.yaml]: (let me try to mkpj)

**Comment by krzyzacy** [comment]
works now https://prow.gflocks.com/log?job=pull-prow-config-validate&id=1101271282147659776

will try to bump cluster and set up report with crier here

**Comment by krzyzacy** [comment]
tbr - will add a tide probably next

**Pull request merged**